templates/guide_fm.html

```diff
@@ -1,73 +1,104 @@
+    we strongly encourage you to participate. You can find more information at
+    <a href="https://studentlife.mit.edu/fyre">studentlife.mit.edu/fyre</a>.
+  </p>
 
-    <h4>DormCon</h4>
-    <p>The Dormitory Council, or DormCon, is a student government body formed by the dormitory presidents to represent
-        dorm residents and facilitate interdorm events. You’re now a dorm resident, so DormCon represents you! Learn
-        more
-        about DormCon and how to get involved at <a href="https://dormcon.mit.edu">dormcon.mit.edu</a>, and
-        add yourself to <a href="https://groups.mit.edu/webmoira/list/dormcon-announce">dormcon-announce@mit.edu</a> to
-        get
-        invited to meetings. Enjoy the booklet we made!</p>
+  <h4>DormCon</h4>
+  <p>
+    The Dormitory Council, or DormCon, is a student government body formed by
+    the dormitory presidents to represent dorm residents and facilitate
+    interdorm events. You're now a dorm resident, so DormCon represents you!
+    Learn more about DormCon and how to get involved at
+    <a href="https://dormcon.mit.edu">dormcon.mit.edu</a>, and either add
+    yourself to the
+    <a href="https://groups.mit.edu/webmoira/list/dormcon-announce"
+      >dormcon-announce mailing list</a
+    >
+    or email
+    <a href="mailto:dormcon-secretary@mit.edu">dormcon-secretary@mit.edu</a> to
+    get invited to meetings. Enjoy the booklet we made!
+  </p>
 
-    <h4>How to Make the Most of This Guide</h4>
-    <p>There is no wrong way to do REX. Plan it out carefully! Mark a couple fun-looking events and play the rest by
-        ear! Wander aimlessly around campus until an upperclassman drags you into their dorm for a card game! Hit every
-        single food event on the list! However you spend these first few days at MIT, remember that now is the time to
-        make friends and make memories. When in doubt, heed the wise words of Ms. Frizzle: "Take chances, make mistakes, and
-        get messy!"
-    </p>
-    <p>With that in mind, here's a quick guide to the guide:</p>
-   
-    <div class="event mandatory">
-        <span class="title">Mandatory Event</span>
-        <span class="description">Events that look like this are mandatory. Make sure you go to them, or
-            you'll miss important information!</span>
-    </div>
-   
+  <h4>How to Make the Most of This Guide</h4>
+  <p>
+    There is no wrong way to do REX. Plan it out carefully! Mark a couple
+    fun-looking events and play the rest by ear! Wander aimlessly around campus
+    until an upperclassman drags you into their dorm for a card game! Hit every
+    single food event on the list! However you spend these first few days at
+    MIT, remember that now is the time to make friends and make memories. When
+    in doubt, heed the wise words of Ms. Frizzle: "Take chances, make mistakes,
+    and get messy!"
+  </p>
+  <p>With that in mind, here's a quick guide to the guide:</p>
+
+  <div class="event tag-mandatory">
```

Comment:
If we aren't going to have mandatory events (like orientation) included, it might be good to comment this out again, but we can figure that out later.

Reply:
It might be good to ask OFY if they want orientation events in the booklet then, I know it's been done in the past so it can't hurt

Reply:
I added if statements so that if there aren't any mandatory (or signature) tags it'll hide that part of the guide